packages/app-store/routing-forms/components/getServerSidePropsSingleForm.ts

```diff
@@ -30,16 +34,6 @@ export const getServerSidePropsForSingleFormView = async function getServerSideP
     };
   }
 
-  const isFormCreateEditAllowed = (await import("../lib/isFormCreateEditAllowed")).isFormCreateEditAllowed;
-  if (!(await isFormCreateEditAllowed({ userId: user.id, formId, targetTeamId: null }))) {
-    return {
-      redirect: {
-        permanent: false,
-        destination: "/403",
-      },
-    };
-  }
-
   const form = await prisma.app_RoutingForms_Form.findUnique({
```

Comment:
I've removed this code block. I think it's no longer necessary because below these lines, we return if this user is able to create or edit Routing form. And the flags will be used in the UI and the actual `formMutation.handler.ts` will finally check if user can do it or not. So I think this create or edit permission check is not really needed here.

What do you think? @hariombalhara